odl/tomo/data/clintec_spect.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Helpers for reading CLINTEC SPECT data formats.
+
+For the bilinear relationship of linear attenuation coefficients and
+Houndsfield units see [Bro+2008]_ and references therein, for example.
+
+"""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import numpy as np
+import dicom
+import os
+
+
+def read_clintec_CT_reconstruction(path):
+    """Reads the data in DICOM format and returns a HU CT volume.
+
+    Parameters
+    ----------
+    path : `str`
+        a path to the folder where DICOM data is
+
+    Returns
+    -------
+    ct_volume : `numpy.ndarray`
+        an array of CT reconstruction in Houndsfield units.
+    dataset : `DICOM` File Meta Information
+
+    """
+
+    dcm_file_list = []
+    for dirName, subdirList, fileList in os.walk(path):
+        for filename in sorted(fileList):
```

Comment:
This could be in the "wrong" order:

```python
>>> sorted(['a1', 'a2', 'a10', 'a11', 'a20'])
['a1', 'a10', 'a11', 'a2', 'a20']
```

I suppose that was why you used `natsort`?